admin/scalafmt.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# set -x
+
+HERE="`dirname $0`"
+VERSION="1.5.1"
+COURSIER="$HERE/.coursier"
+SCALAFMT="$HERE/.scalafmt-$VERSION"
+
+if [ ! -f $COURSIER ]; then
+  curl -L -o $COURSIER https://git.io/vgvpD
+  chmod +x $COURSIER
+fi
+
+if [ ! -f $SCALAFMT ]; then
+  $COURSIER bootstrap com.geirsson:scalafmt-cli_2.11:$VERSION --main org.scalafmt.cli.Cli -o $SCALAFMT
+  chmod +x $SCALAFMT
+fi
+
+$SCALAFMT "$@"
```

Comment:
Why is it useful to run scalafmt on the CI?

Reply:
It's to make sure you don't push code that is not formatted. I create a separate matrix entry so you know that the CI failed only because the code was not formatted, and not more serious test failures.